client.go

```diff
@@ -139,7 +151,9 @@ func (c *Client) GetMap(ctx context.Context, name string) (*Map, error) {
 		// there is a near cache config for this map
 		ncmgr := c.getNearCacheManager(ServiceNameMap)
 		nc := ncmgr.GetOrCreateNearCache(name, ncc)
-		m.ncm, err = newNearCacheMap(nc, &ncc, c.ic.SerializationService)
+		ss := c.ic.SerializationService
+		rt := ncmgr.RepairingTask()
```

Comment:
As far as I see in Java only one repairing task is created per data structure type. Can we do the same?  see https://github.com/hazelcast/hazelcast/blob/14ef0dbefa4a6fa00a50a8873aeea02ec6c3f5e4/hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTask.java#L45 and https://github.com/hazelcast/hazelcast/blob/14ef0dbefa4a6fa00a50a8873aeea02ec6c3f5e4/hazelcast/src/main/java/com/hazelcast/client/impl/spi/ClientContext.java#L117

Reply:
As far as I can trace, go client can create multiple repair tasks for map data type. Am I wrong 🤔

Reply:
Maybe we should pass the service name to `GetOrCreateNearCache` to replicate the same behaviour in `nearcache/manager.go`

Reply:
> It's the same with the Go client. Currently only one repairing task is created when the near cache manager is initialized.

You are right. 

> As far as I can trace, go client can create multiple repair tasks for map data type

Can you share the scenario? I thought a new reparing task is being created in this line but it is not. 

so, resolving the comment